ujt/app.py

```diff
@@ -71,22 +76,35 @@ def read_local_data() -> Tuple[Dict[str, Node], Dict[str, Client]]:
 
     node_names = list(service_names) + flattened_endpoint_names
 
-    node_name_message_map: Dict[str, Node] = {
-        name: cast(
-            Node,
-            utils.read_proto_from_file(
-                utils.named_proto_file_name(name, Node),
-                Node,
-            )) for name in node_names
-    }
-
-    client_name_message_map: Dict[str, Client] = {
-        name: cast(
-            Client,
-            utils.read_proto_from_file(
-                utils.named_proto_file_name(name, Client), Client))
-        for name in client_names
-    }
+    node_name_message_map: Dict[str,
+                                Node] = {
```

Comment:
is this really what the linter did? one argument on each line seems excessive

Reply:
I messed with some settings... I think if a line breaks the length limit it'll break on every open paren and comma? I'm still not happy with the formatting settings so I plan on adding [black](https://github.com/psf/black) eventually.